source/ios/tools/AdaptiveCards.podspec

```diff
@@ -42,6 +42,6 @@ Pod::Spec.new do |spec|
 
   spec.exclude_files = 'source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/include/**/*'
 
-  spec.dependency 'MicrosoftFluentUI', '~> 0.3.6'
```

Comment:
@jwoo-msft is this correct? does it only rely on **toolip** from **fluentUI** library?

Reply:
@beervoley, that's correct.